content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -247,6 +247,287 @@ EOF
 {{< /tab >}}
 {{< /tabset >}}
 
+## Configure telemetry to be cluster-aware
+
+In multi-cluster scenarios, being able to identify traffic by source and destination clusters
+can be useful. Istio has two built-in labels for holding cluster information. These labels
+can be enabled with a few changes to the metrics configuration.
+
+Modify the Istio metrics configuration  **in each cluster that you are adding to the mesh** to enable source and
+destination cluster dimension.
+
+1. Copy the configuration for HTTP and TCP metrics to local disk.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system get envoyfilter stats-filter-1.6 -o yaml > stats-filter-1.6.yaml
+    $ kubectl -n istio-system get envoyfilter tcp-stats-filter-1.6 -o yaml > tcp-stats-filter-1.6.yaml
+    {{< /text >}}
+
+1. Replace the existing `spec` sections of **both** of those files, with the following content:
```

Comment:
@douglas-reid this seems a bit heavy-handed for the install guide. Is there a way we can make this simpler? Could we ship a script with the samples, for example?

FYI @rcaballeromx